brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1429 Improvement: Copy Paste support to general tab controls

### Description
Add copy-paste style support to needful controls from the general block.

Currently, we support only style tab controls, we need to keep the needful settings from the general tab too.

Here is a sheet with the list of blocks and the controls for which support is added - https://docs.google.com/spreadsheets/d/1UuXcnjX--KJqopjtfW-BET5kNesBwfZvUeGGXkzs7C4/edit?usp=sharing

### Screenshots


### Types of changes




Improvement

### How has this been tested?
- Drag drop blocks
- Apply styles from general tab
- After pasting style all the needful style should be applied

### Checklist:
